docs/source/conf.py

```diff
@@ -17,6 +17,9 @@
 # is relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
 sys.path.append(os.path.abspath('sphinxext'))
+sys.path.append(os.path.abspath(os.path.join('..','..')))
+
+from chaco import __version__
```

Comment:
I think the idea is for chaco to be installed before building the docs. It shouldn't be necessary to monkey with `sys.path`